--- pangopdf-1.2.3.6/pango/opentype/pango-ot-info.c.fix	2004-08-09 11:27:58.535153008 +0200
+++ pangopdf-1.2.3.6/pango/opentype/pango-ot-info.c	2004-08-09 11:28:07.124847176 +0200
@@ -22,7 +22,6 @@
 #include "pango-ot-private.h"
 #include "fterrcompat.h"
 #include <freetype/internal/ftobjs.h>
-#include <freetype/ftmodule.h>
 
 static void pango_ot_info_class_init (GObjectClass *object_class);
 static void pango_ot_info_finalize   (GObject *object);
--- pangopdf-1.2.3.6/pango/opentype/pango-ot-private.h.fix	2004-08-09 11:28:25.563044144 +0200
+++ pangopdf-1.2.3.6/pango/opentype/pango-ot-private.h	2004-08-09 11:28:57.012263136 +0200
@@ -22,6 +22,8 @@
 #ifndef __PANGO_OT_PRIVATE_H__
 #define __PANGO_OT_PRIVATE_H__
 
+#include <ft2build.h>
+#include FT_FREETYPE_H
 #include <freetype/freetype.h>
 
 #include <glib-object.h>
--- pangopdf-1.2.3.6/pango/opentype/ftxopen.h.fix	2004-08-09 11:29:43.729161088 +0200
+++ pangopdf-1.2.3.6/pango/opentype/ftxopen.h	2004-08-09 11:29:57.449075344 +0200
@@ -21,7 +21,8 @@
 
 #ifndef FTXOPEN_H
 #define FTXOPEN_H
-
+#include <ft2build.h>
+#include FT_FREETYPE_H
 #include <freetype/freetype.h>
 
 #ifdef __cplusplus
--- pangopdf-1.2.3.6/pango/pango-ot.h.fix	2004-08-09 11:33:22.800857128 +0200
+++ pangopdf-1.2.3.6/pango/pango-ot.h	2004-08-09 11:33:46.815206392 +0200
@@ -21,7 +21,8 @@
 
 #ifndef __PANGO_OT_H__
 #define __PANGO_OT_H__
-
+#include <ft2build.h>
+#include FT_FREETYPE_H
 #include <freetype/freetype.h>
 #include <pango/pango-glyph.h>
 
--- pangopdf-1.2.3.6/pango/pangoft2.c.fix	2004-08-09 11:35:01.918788912 +0200
+++ pangopdf-1.2.3.6/pango/pangoft2.c	2004-08-09 11:35:20.164015216 +0200
@@ -28,6 +28,8 @@
 #include <glib.h>
 #include <glib/gprintf.h>
 
+#include <ft2build.h>
+#include FT_FREETYPE_H
 #include <freetype/freetype.h>
 
 #include "pango-utils.h"
--- pangopdf-1.2.3.6/pango/pangoft2.h.fix	2004-08-09 11:35:46.007086472 +0200
+++ pangopdf-1.2.3.6/pango/pangoft2.h	2004-08-09 11:36:34.430724960 +0200
@@ -23,6 +23,8 @@
 #ifndef __PANGOFT2_H__
 #define __PANGOFT2_H__
 
+#include <ft2build.h>
+#include FT_FREETYPE_H
 #include <freetype/freetype.h>
 
 #include <fontconfig/fontconfig.h>
--- pangopdf-1.2.3.6/modules/indic/indic-ot.h.fix	2004-08-09 11:40:20.253394696 +0200
+++ pangopdf-1.2.3.6/modules/indic/indic-ot.h	2004-08-09 11:41:15.478999128 +0200
@@ -9,6 +9,8 @@
 #ifndef __INDIC_OT_H__
 #define __INDIC_OT_H__
 
+#include <ft2build.h>
+#include FT_FREETYPE_H
 #include <freetype/freetype.h>
 #include <pango/pango-glyph.h>
 #include <pango/pango-types.h>
